src/Layout/helper.js

```diff
@@ -0,0 +1,12 @@
+import { Children } from 'react'
+import { LayoutTopBar } from './LayoutTopBar'
+import { LayoutSidebar } from './LayoutSidebar'
+
+export const childrenContainsComponent = component => children =>
+    Children.toArray(children).reduce(
+        (found, child) => found || child.type === component,
+        false
+    )
```

Comment:
just returning this would a bit easier on the eye:
```javascript
!!Children.toArray(children).find(({ type }) => type === component)
```
(and less iterations, but that's not really relevant in this context...)